Aztec/Classes/Public/TextKit/AztecTextStorage.swift

```diff
@@ -245,7 +261,7 @@ public class AztecTextStorage: NSTextStorage {
         return html
     }
 
-    public func setHTML(html: String, withDefaultFontDescriptor defaultFontDescriptor: UIFontDescriptor) {
+    func setHTML(html: String, withDefaultFontDescriptor defaultFontDescriptor: UIFontDescriptor) {
```

Comment:
To be honest I don't think it should.  Our text storage is not designed to be used directly, but only through `Aztec.TextView`, which has the necessary interface for setting the HTML.